### PR TITLE
modified prizes

### PR DESCRIPTION
**Pull request by ishikajn8126**

### CODE REVIEW

**Review by aaryanporwal — APPROVED** [review]
lgtm

**Comment by aaryanporwal** [comment]
Hey @ishikajn8126 i just noticed` !important`. Can you try to remove it?

File: rules.html [hunk 1]
```diff
@@ -133,13 +133,22 @@ <h1>Rules</h1>
       The Organizing Team's decision on conflicts, results, prizes or any other
       dispute would be made in all fairness and stands final.
     </li>
-    
+  </ul>
+  <ul class="rules" style="margin-bottom:2%;" id="oss-track">
+    <li type="none">
+      <h2 style="color:#c71239;">OSS Track</h2>
+    </li>
+    <li>Being really passionate about  Free/Open-Source Software, we have a separate track track
```
Comment (aaryanporwal) [inline rules.html]: typo: `track` used twice

**Review by akshgpt7 — APPROVED** [review]
Great job! Thanks for this.

**Pull request merged**